db/migrate/20220118221746_create_feeds.rb

```diff
@@ -0,0 +1,43 @@
+class CreateFeeds < ActiveRecord::Migration
+  def change
+    remove_column :podcasts, :max_episodes, :integer
+
+    create_table :feeds do |t|
+      t.references :podcast, index: true, foreign_key: true
+      t.string :label
+      t.string :slug, null: false
+
+      # optional overrides
+      t.text :title
+      t.string :url
+      t.string :path
+      t.string :enclosure_prefix
+      t.string :enclosure_template
+      t.integer :display_episodes_count
+      t.integer :display_full_episodes_count
+
+      # optional sub-feed settings
+      t.boolean :private, default: true
+      t.integer :publish_offset_seconds
+      t.text :reject_zones
```

Comment:
Sticking with ruby semantics here - a JSON encoded array of zone ids/types you want to _remove_ from these arrangements.  So by default (when null) - you get everything.  But assuming a bit of magical allowed terms:

```ruby
feed.reject_zones = ["ad"] # all type="ad"
feed.reject_zones = ["house"] # just "house"
feed.reject_zones = ["house", "paid"] # equivalent to "ad"
feed.reject_zones = ["pre_"] # zones starting with "pre_"
feed.reject_zones = ["sonic"]
```

This is a bit ambiguous at the moment, since Dovetail arrangements only have 4 zone "types" (`a` ad, `o` original, `i` sonic_id, and `b` billboard).  I'd like to at least add `h` house to that someday.

But the point is - this is a list of some kind of strings, which DTR knows how to interpret.